cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,613 @@
+            'contains(text(), "Error:")]'
+        )
+        error_icon = Text(locator='.//span[contains(@class, "pficon-error-circle-o")]')
+        popover_text = Text(locator='.//div[contains(@class, "popover-content")]')
+
+        def after_fill(self, was_change):
+            self.browser.send_keys(Keys.ENTER, self.search_box)
+
+        @property
+        def is_displayed(self):
+            return self.table.is_displayed and (
+                len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0
+            )
+
+        def filter(self, type, name):
+            try:
+                self.filter_by_dropdown.item_select(type)
+            except NoSuchElementException:
+                logger.info("{} not present in filter dropdown!".format(type))
+            self.search_box.fill(name)
+            self.after_fill(was_change=True)
+
+        def select_by_name(self, vm_name):
+            self.filter("VM Name", vm_name)
+            vms_selected = []
+            for row in self.table.rows():
+                if vm_name in row.read()["VM Name"]:
+                    row.select.fill(True)
+                    vms_selected.append(row.read()["VM Name"])
+            return vms_selected
+
+    @View.nested
+    class advanced(View):  # noqa
+        pre_playbook = BootstrapSelect("preMigrationPlaybook")
+        post_playbook = BootstrapSelect("postMigrationPlaybook")
+        pre_checkbox = Text(locator='.//input[contains(@id, "pre_migration_select_all")]')
+        post_checkbox = Text(locator='.//input[contains(@id, "post_migration_select_all")]')
+
+        @property
+        def is_displayed(self):
+            return self.pre_playbook.is_displayed
+
+    @View.nested
+    class schedule(View):  # noqa
+        create = Button("Create")
+        run_migration = RadioGroup('.//div[contains(@id,"migration_plan_choice_radio")]')
+
+        @property
+        def is_displayed(self):
+            return self.run_migration.is_displayed
+
+    @View.nested
+    class results(View):  # noqa
+        close = Button("Close")
+        msg = Text('.//h3[contains(@id,"migration-plan-results-message")]')
+
+    @property
+    def is_displayed(self):
+        return self.title.text == self.form_title_text
+
+
+class MigrationPlanRequestDetailsView(View):
+    migration_request_details_list = MigrationPlanRequestDetailsList("plan-request-details-list")
+    sort_type = SelectorDropdown("id", "sortTypeMenu")
+    paginator_view = View.include(V2VPaginatorPane, use_parent=True)
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
+    sort_order = Text(".//button[./span[contains(@class,'sort-direction')]]")
+    filter_by_dropdown = SelectorDropdown("id", "filterFieldTypeMenu")
+    filter_by_status_dropdown = SelectorDropdown("id", "filterCategoryMenu")
+    sort_by_dropdown = SelectorDropdown("id", "sortTypeMenu")
```

Comment:
sort_by_drowndown and sort_type looks to be the same? Was this suppose to be the sort button?

Reply:
Removed